app-info/eos-extra-pwa.yaml

```diff
@@ -79,7 +79,8 @@
   adaptive: true
 - url: https://pinafore.social/
   name_property: name
-  description: An alternative web client for Mastodon, focused on speed and simplicity.
+  summary: An alternative web client for Mastodon, focused on speed and simplicity.
+  description: 
```

Comment:
I think you need to come up with some description here. You also need to run `pwa-metainfo-generator.py` again and commit the changes to the metainfo files.